Add example for python unittest annottations (#340)
cirruslabs/cirrus-ci-docs: docs/examples.md

```yaml
container:

>   image: python:slim

Reviewer: Why not using the default image?

Author: Because it is smaller, meaning the tests start faster

  script: flake8 *.py
```

### `Unittest` Annotations

Python Unittest reports are supported by [Cirrus CI Annotations](https://medium.com/cirruslabs/github-annotations-support-227d179cde31).
This way you can see what tests are failing without leaving the pull request you are reviewing! Here is an example
of `.cirrus.yml` that produces and stores `Unittest` reports:

> 

Author: ```suggestion
```

> 

Author: ```suggestion
```

  script: flake8 *.py
```

### `Unittest` Annotations

Python Unittest reports are supported by [Cirrus CI Annotations](https://medium.com/cirruslabs/github-annotations-support-227d179cde31).
This way you can see what tests are failing without leaving the pull request you are reviewing! Here is an example
of `.cirrus.yml` that produces and stores `Unittest` reports:

> This can be set up by doing the following:

Reviewer: ```suggestion
```

> Add this task to your `.cirrus.yml`:

Reviewer: Too many colons and this is not necessary now

```suggestion
```

```yaml
unittest_task:

>   # feel free to rename the task

Reviewer: Let's remove

```suggestion
```

  container:
    image: python:slim
  install_dependencies_script: |
    pip3 install unittest_xml_reporting
  run_tests_script: python3 -m xmlrunner tests
  # replace 'tests' with the module,
  # unittest.TestCase, or unittest.TestSuite
  # that the tests are in
  always:
    upload_results_artifacts:
      path: ./*.xml
      format: junit

>       # yes, it must say junit to work

Reviewer: ```suggestion
```

```

> And you should get annotations for your test results.

Reviewer: ```suggestion
Now you should get annotations for your test results.
```

  script: flake8 *.py
```

### `Unittest` Annotations

> Cirrus CI annotations are available for Python Unittest results. This way you can see what tests are failing without leaving the pull request you are reviewing!

Reviewer: What about a short description like this:

```suggestion
Python Unittest reports are supported by [Cirrus CI Annotations](https://medium.com/cirruslabs/github-annotations-support-227d179cde31).
This way you can see what tests are failing without leaving the pull request you are reviewing! Here is an example
of `.cirrus.yml` that produces and stores `Unittest` reports:
```

This can be set up by doing the following:

> First, add tests if you haven't already. Here is a basic one that just makes sure you are running it from Cirrus CI:

Reviewer: Not sure if people already reading about Unittest don't have tests already. 🤔

Author: True.

```python
import unittest

class Tests(unittest.TestCase):
    def setUp(self):
        self.is_ci = os.getenv("CI") != None

    def test_env(self):
        self.assertTrue(self.is_ci)

if __name__ == '__main__':
    unittest.main()
```

Next, add this to your `.cirrus.yml`:

```yaml

> unittest_with_cool_annotations_task:

Reviewer: How about simply?

```suggestion
unittest_task:
```
